storagenode/storagenodedb/pieceinfo.go

```diff
@@ -18,28 +18,34 @@ import (
 	"storj.io/storj/storagenode/pieces"
 )
 
+// ErrPieceInfo represents errors from the piece info database.
+var ErrPieceInfo = errs.Class("v0pieceinfo error")
+
 type v0PieceInfo struct {
```

Comment:
nit - maybe rename to `v0PieceInfoDB` to be consistent with the other db name formatting.. but also this doesn't seem like a dealbreaker

Reply:
Done!